interfaces/builtin/spi.go

```diff
@@ -84,9 +84,15 @@ func (iface *spiInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 		return nil
 	}
 	spec.AddSnippet(fmt.Sprintf("%s rw,", path))
+
+	spiBusNum := strings.Split(strings.TrimPrefix(path, "/dev/spidev"), ".")[0]
+	spec.AddParametricSnippet([]string{
+		"/sys/devices/platform/**.spi/**/spi" /* ###PARAM### */, "/** rw,",
```

Comment:
I think we're still expecting at least one level of nesting under platform, usually some `bus*`, or `soc*` entry
```suggestion
		"/sys/devices/platform/**/**.spi/**/spi" /* ###PARAM### */, "/** rw,",
```